build/main.mk

```diff
@@ -19,7 +19,7 @@
 # you might have to be more specific, for example "python2" or "python2.6".
 # Or if the Python interpreter is not in the search path, you can specify its
 # full path.
```

Comment:
We might want to update this comment as well, as it makes no sense anymore to use python2 with Python 3 specific code.

Reply:
Good point. I've mostly just addressed compile errors. There may be more places where the compile doesn't fail that for some reason need to be adjusted. I've force-pushed the change to my fork, so it should be fixed when it gets merged.